docs/REST API/rest-api.md

```diff
@@ -10302,6 +10147,7 @@ Content-Type: application/octet-stream
 {
   "software_package": {
     "name": "FalconSensor-6.44.pkg",
+    "display_name": "",
```

Comment:
@jahzielv 
I only see display_name returned as a top level key and not in `software_package`? Did I miss a design decision or do we need to add it?

<img width="1115" height="929" alt="Image" src="https://github.com/user-attachments/assets/0b5a4a15-94b0-4c41-8ab8-06c6b9605c1c" />